src/app/AppReducer.js

```diff
@@ -11,21 +11,11 @@ const initialState = {
 const app = (state = initialState, action) => {
   switch (action.type) {
 
-  case types.CONFIG_REQUEST:
-    return {...state, loadingConfig: true};
-
-  case types.CONFIG_LOADED:
-
-    // whoa! this is a stopgap until I write the xenia package.
-    // if I don't do this, I'd have to implement some insane redux middleware
-    // only to unwind it after the xenia package anyway.
-    window.basicAuthorization = action.config.basicAuthorization;
-    types.configXenia(action.config);
-
-    return {...state, loadingConfig: false, configLoaded: true, ...action.config};
+  /* put some app-level actions here. probably error reporting */
```

Comment:
Unclear at this point what the role of this reducer is - there's no loading state or data being stored in it, correct? Can this be removed?